.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,77 @@
+name: Bug report
+description: Create a report to help us improve
+labels: ["bug"]
```

Comment:
Maybe a good opportunity to address #14014 too by adding `needs-triage` here (and in the other templates)?

Reply:
I'll add that. I think one thing to call out from the discussion in #14014 is that if someone opens an issue without using one of these templates, an action would be needed to apply the `needs-triage` label.

Reply:
All of the labels appear to use spaces to delimit words instead of hyphens, so I added it as `needs triage` for now. I think we should be consistent here, so if we would prefer to use hyphens, let me know and I'll make a separate effort to convert all the labels to hyphens.

Reply:
Please take a look at https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14103 as a resolution to the issue of a blank template. That PR disables the blank template and adds a custom blank template called "other issue"

Reply:
I am fine with using spaces if that's what we usually use, I didn't think much about it when proposing the specific label name 😅